### ISSUE TITLE
Could not pass event EntityDeathEvent to MobDrops + spigot jar differs from github code

### ISSUE THREAD

**Issue by gg4goat** [ScillDev/MobDrops#1]
[17:29:00 ERROR]: Could not pass event EntityDeathEvent to MobDrops v1.0
java.lang.NullPointerException: null
        at me.scill.mobdrops.RandomCollection.next(RandomCollection.java:26) ~[?:?]
        at me.scill.mobdrops.listeners.DropsListener.dropLoot(DropsListener.java:37) ~[?:?]
        at me.scill.mobdrops.listeners.DropsListener.onDeath(DropsListener.java:27) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor90.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.4.jar:git-Paper-288]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.4.jar:git-Paper-288]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.4.jar:git-Paper-288]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.4.jar:git-Paper-288]
        at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:801) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityLiving.d(EntityLiving.java:1491) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityLiving.die(EntityLiving.java:1420) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityLiving.damageEntity(EntityLiving.java:1262) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityMonster.damageEntity(EntityMonster.java:52) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityZombie.damageEntity(EntityZombie.java:250) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.Entity.entityBaseTick(Entity.java:449) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityLiving.entityBaseTick(EntityLiving.java:266) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityInsentient.entityBaseTick(EntityInsentient.java:243) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.Entity.tick(Entity.java:406) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityLiving.tick(EntityLiving.java:2430) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityInsentient.tick(EntityInsentient.java:306) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.EntityZombie.tick(EntityZombie.java:173) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.WorldServer.entityJoinedWorld(WorldServer.java:845) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.World.a(World.java:884) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.WorldServer.doTick(WorldServer.java:530) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1359) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1000) ~[patched_1.16.4.jar:git-Paper-288]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-288]
        at java.lang.Thread.run(Thread.java:834) [?:?]

I am not seeing me.scill.mobdrops.RandomCollection on this repo

**Comment by ColinGrime** [comment]
Sorry for the confusion -- I never updated the jar because I only changed a few names.  I'll be revising the code for this project soon, so I'll have it updated shortly.  In the meanwhile, could you send over the config.yml file you've created?

**Comment by gg4goat** [comment]
Sorry I discarded the file, but it was something basic with a few

item:
  - 1 0
  - 2 0 
  - 3 0

**Comment by ColinGrime** [comment]
The format for the loot is [amount] [percent].  So in your example, you are giving 0% for 1 drop, 0% for 2 drops, and 0% for 3 drops.  So yes, an error was occurring when every item chance is at 0%.

I've revised a good portion of the code in the plugin while also allowing completely empty drops.  I'm sure you didn't mean to have it completely empty, but nevertheless, I appreciate you finding the bug.  I've tested the plugin on 1.16.4 & submitted an update on Spigot.
